src/schema/bookmarks.ts

```diff
@@ -21,6 +21,10 @@ import {
 import { SelectQueryBuilder } from 'typeorm';
 import { GQLPost } from './posts';
 import { Connection } from 'graphql-relay';
+import {
```

Comment:
It's better to manage the workflows on the CDC side of things. It's much more fault tolerant and you also get the before and after values for easily canceling existing workflows. Should make everything a little bit simpler

Reply:
Yeah, thought of that too at first but did not push through. Let's do it.